src/me/TomTheDeveloper/InvasionInstance.java

```diff
@@ -595,9 +603,10 @@ public void startWave() {
         for (User user : UserManager.getUsers(this)) {
             user.getKit().reStock(user.toPlayer());
         }
-
-        getChatManager().broadcastMessage("Wave-Started", "Wave " + ChatManager.HIGHLIGHTED + "%NUMBER%" + ChatColor.GRAY + " started!", wave);
-
+        String message = ChatManager.formatMessage(ChatManager.colorMessage("Wave-Started"), wave);
+        for(Player player1 : getPlayers()) {
+            player1.sendMessage("�a[Village Defense] " + message);
```

Comment:
Don't hardcode [Village Defense] in the code since not everybody wants this tag. They want to be able to translate it or even remove it.

Reply:
oh i didn't saw it, i'm getting prefix from ChatManager class but i didn't changed that :p